### PR TITLE
SE-935-Remove-AppManager-References-from-Deployment-Test-Scripts

### PR DESCRIPTION
**Pull request by ShaneDuncan602**
…ployments with the DeployExampleApplication.sh

### CODE REVIEW

File: deployAppERC721Test.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@ GREEN='\033[0;32m'
 YELLOW='\033[0;33m'
 NC='\033[0m' # No Color
 
-./foundry-version-check.sh
+# ./foundry-version-check.sh
```
Comment (VoR0220) [inline deployAppERC721Test.sh]: Do we get rid of this? If not, maybe we can get a comment?

**Review by VoR0220 — APPROVED** [review]
Just one small thing and other than that, this looks good to go.

File: deployAppERC721Test.sh (hunk 1 above)
Comment (ShaneDuncan602) [reply deployAppERC721Test.sh]: Nope. Good catch. This should not be commented out.

**Review by TJ-Everett — APPROVED** [review]
Looks great! Approved

**Pull request merged**